### PR TITLE
fix: MSB-973, MSB-974 fix configureLogging and selfTest parsers

### PR DESCRIPTION
**Pull request by maximsk**
fix `selfTest` and `configureLogging`
 - [x] fix tag number of `selfTest.result`
 - [x] fix tag number of `configureLogging.result` 
 - [x] implement parsing of `configureLogging.parameters` 
 - [x] add test for `ConfigureLoggingSystemLogMessage`
Documentation: https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Publikationen/TechnischeRichtlinien/TR03153/TR-03153_Anwendungshinweise.pdf?__blob=publicationFile&v=1

### CODE REVIEW

**Review by kplakidas — APPROVED** [review]

**Review by williambaer — APPROVED** [review]

File: src/main/resources/ttc.properties [hunk 1]
```diff
@@ -76,10 +76,9 @@ de.konfidas.ttc.messages.systemlogs.errorDescriptionNotFound = Fehler beim Parse
 de.konfidas.ttc.messages.systemlogs.errorDeviceInformationSetEndedEarly = DeviceInformationSet in updateTime endete vorzeitig.
 de.konfidas.ttc.messages.systemlogs.errorDeviceInformationSetOfUpdateTimeEndedEarly = DeviceInformationSet in updateTime endete vorzeitig.
 de.konfidas.ttc.messages.systemlogs.errorEarlyEndOfSystemOperationData = Fehler beim Parsen des systemOperationDataContent. Vorzeitiges Ende von systemOperationData.
-de.konfidas.ttc.messages.systemlogs.errorLoggingParameterNotOfTypeASN1Sequence = Fehler beim Parsen des loggingParameter. Der Eintrag sollte vom Typ ASNSequence sein.
+de.konfidas.ttc.messages.systemlogs.errorParametersDoesNotStartWithOctetString = Fehler beim Parsen des loggingParameter. Der Eintrag sollte vom Typ ASN1OctetString sein.
```
Comment (kplakidas) [inline src/main/resources/ttc.properties]: -> `Fehler beim Parsen des loggingParameter. Das Pflichtfeld loggingParameter startet nicht mit einem Octet String.` for consitency

File: src/main/resources/ttc.properties [hunk 2]
```diff
@@ -94,6 +93,9 @@ de.konfidas.ttc.messages.systemlogs.logoutCauseNotFound = Fehler beim Parsen des
 de.konfidas.ttc.messages.systemlogs.roleNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld role wurde nicht gefunden.
 de.konfidas.ttc.messages.systemlogs.timeAfterUpdateNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichtfeld timeAfterUpdate wurde nicht gefunden
 de.konfidas.ttc.messages.systemlogs.timeBeforeUpdateNotFoud = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil timeBeforeUpdate wurde nicht gefunden
```
Comment (kplakidas) [inline src/main/resources/ttc.properties]: Pflichteil -> Pflichtfeld (also in the subequent ones)

File: src/main/resources/ttc.properties [hunk 3]
```diff
@@ -94,6 +93,9 @@ de.konfidas.ttc.messages.systemlogs.logoutCauseNotFound = Fehler beim Parsen des
...
+de.konfidas.ttc.messages.systemlogs.errorEventNameNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil eventName wurde nicht gefunden
+de.konfidas.ttc.messages.systemlogs.errorEnabledNotFound = Fehler beim Parsen des systemOperationDataContent. Das Pflichteil enabled wurde nicht gefunden
+de.konfidas.ttc.messages.systemlogs.errorLoggingParameterUnknownTag = Fehler beim Parsen des systemOperationDataContent. Das Tag ist unbekannt
```
Comment (kplakidas) [inline src/main/resources/ttc.properties]: errorLoggingParameterUnknownTag != systemOperationDataContent (something is wrong here)

**Review by kplakidas — CHANGES_REQUESTED** [review]

File: src/main/resources/ttc.properties (hunk 2 above)
Comment (kplakidas) [reply src/main/resources/ttc.properties]: Ermmm, not done. Lines 94, 95 as well.

**Review by kplakidas — CHANGES_REQUESTED** [review]

File: src/main/resources/ttc.properties (hunk 2 above)
Comment (maximsk) [reply src/main/resources/ttc.properties]: done, code is outdated

**Review by kplakidas — APPROVED** [review]

**Pull request merged**